### DISCUSSION TITLE
What does democratizing pipelines mean?

### DISCUSSION THREAD

**Discussion by liutiming** [ropensci/targets#284] [General]
https://github.com/ropensci/targets/issues/282#issuecomment-767917921

**Comment by wlandau** [comment]
Great question. I think it means making pipelines accessible, convenient, and useful to a large audience. R Markdown has achieved this, so much so that people use it in situations where they would be better off using pipelines. I am trying to close the gap between need and adoption.

The two approaches I have tried so far are

1. Teach people how to speak function:
    * https://books.ropensci.org/targets/functions.html
    * https://github.com/wlandau/targets-tutorial/blob/main/1-functions.Rmd
2. Start to build the R Targetopia: https://wlandau.github.io/targetopia/

**Reply by liutiming** [reply]
great reply. thanks!
